### ISSUE TITLE
Display interactive hex value in `color_picker`

### ISSUE THREAD

**Issue by darshanCommits** [woelper/oculante#578]
Kind of odd that it has color picker, RGB values but not hex.

**Comment by woelper** [comment]
> Kind of odd that it has color picker, RGB values but not hex.
> 

I am not sure I understand this correctly: do you wish to enter the background colour as hex value? And by background colour you mean the configuration in settings?

**Comment by darshanCommits** [comment]
> I am not sure I understand this correctly: do you wish to enter the background colour as hex value? And by background colour you mean the configuration in settings?

Yes, in the configuration. I only found out about this app today so im unsure if it is lacking anywhere else as my usage is pretty basic.

![image](https://github.com/user-attachments/assets/a569b95f-1fe2-4f4f-904b-0b5adb05537d)

**Comment by Stoppedpuma** [comment]
> Yes, in the configuration. I only found out about this app today so im unsure if it is lacking anywhere else as my usage is pretty basic.

Same applies to everywhere with the colour picker. I'm pretty sure the reason this wasn't an option out of the gate is that it's not an option [on the egui `color_picker`](https://docs.rs/egui/latest/egui/widgets/color_picker/index.html).

**Comment by woelper** [comment]
Doing this would require writing our own color picker, which is not a priority, at least for me. I think this could be a nice area for a contribution. The color picker could either done as a direct contribution to egui, or as a separate crate.

**Comment by Stoppedpuma** [comment]
I was afraid that would be the case, I'm in no particular rush for this either so I'm going to remove this from the current milestone and release plan.

**Comment by woelper** [comment]
I am closing this as there is now a hex display in the info panel.

**Comment by Stoppedpuma** [comment]
> I am closing this as there is now a hex display in the info panel.

I'll keep this closed for you to decide if this is something worth re-opening for, but I think this is more a UX thing more than anything. It would be nice to be able to interact with the hex or whatever other colour type the user would prefer with options like copy + paste and what not. This was what the user was requesting I believe, not so much being able to view the hex of the hovered colour on the info panel (though this is great to have as well).

**Comment by woelper** [comment]
Ok, got it, thanks! Maybe we should open a new issue with a custom color picker then? 
I know I said that this was not highest prio for me, but I am thinking that having a nice modular color picker that could be used in other places could be beneficial. I like the way some color grading tools have just color wheels that let you quickly adjust. Maybe we can construct a  color picker from base elements like a color wheel which we can re-use in other parts of the UI?

**Comment by Stoppedpuma** [comment]
I'll open a separate issue then as I've ran into this several times and think this is pretty huge to have in relevant areas.
